cypress/e2e/upload-download.cy.ts

```diff
@@ -0,0 +1,40 @@
+import {UploadFilePage, DownloadFilePage} from "../page/index";
+
+describe("Upload and download Files", () => {
+  let uploadPage: UploadFilePage;
+  let downloadFilePage: DownloadFilePage;
+
+  before(() => {
+    uploadPage = new UploadFilePage();
+    downloadFilePage = new DownloadFilePage();
+  });
+
+  describe("Upload a file", () => {
+    it("Should have a Tittle", () => {
+      // Arrange
+      const fileName = "example.json";
+      uploadPage.visitUploadFilePage();
+
+      // Act
+      uploadPage.uploadFile(fileName);
+      uploadPage.getTittle(fileName);
+
+      // Assert
+      uploadPage.verifyTittle(fileName);
```

Comment:
decide between getTitle or verifyTitle you can't have both. and if it is a get, why it receives a parameter?

Reply:
I understand the get logic, but, what function does call the get if you are already calling in the verify, it is very extrange call a get and don't assign the value or perform another action based on the value that the method returns

Reply:
resolved